modules/@apostrophecms/schema/index.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
         // This is new and until now if JS client side failed, then it would
         // allow the save with empty values -Lars
         if (field.required && (_.isUndefined(data[field.name]) || !data[field.name].toString().length)) {
-          throw self.apos.error('required');
+          throw self.apos.error(`required: ${field.name}`);
```

Comment:
`'required'` is a specific response that the tests look for. You can add a more helpful message as a second argument, like here: https://github.com/apostrophecms/apostrophe/blob/3/palette-prep/modules/@apostrophecms/attachment/index.js#L433

Reply:
oo didn't mean to commit this, for my own debugging